evalml/pipelines/utils.py

```diff
@@ -1486,37 +1489,61 @@ def stack_X(X, series_id_name, time_index, starting_index=None):
         time_index (str): The name of the time index column.
         starting_index (int): The starting index to use for the stacked DataFrame. If None, the starting index
             will match that of the input data. Defaults to None.
+        series_id_values (set): The unique values of a series ID, used to generate the index. If None, values will
```

Comment:
Can you add a check to make sure `series_id_values` is a set? I have a feeling it'll be a common error to pass in a list instead. Unless it's not actually necessary for it to be a set instead of a list, in which case we could mention both here.

Reply:
sure, updated to clarify it can also be a list as well.